easybuild/easyconfigs/c/CmdStanR/CmdStanR-0.5.2-foss-2022a-R-4.2.1.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'RPackage'
+
+name = 'CmdStanR'
+version = '0.5.2'
+versionsuffix = '-R-%(rver)s'
+
+homepage = 'https://mc-stan.org/cmdstanr'
+description = "CmdStanR is a lightweight interface to Stan for R users"
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+
+sources = [{
+	'filename': 'cmdStanR-%(version)s.tar.gz',
+	'git_config': {
+		'url': 'https://github.com/stan-dev',
+		'repo_name': 'cmdstanr',
+		'tag': 'v%(version)s'}}]
+checksums = ['ea5cd77c232f27a879be83721ed379a08283140f4696d67278ce2087784c10b3']
```

Comment:
using `git_config` means that we don't get a stable checksum. Instead, we can use:
```
source_urls = ['https://github.com/stan-dev/cmdstanr/archive/refs/tags/']
sources = ['v%(version)s.tar.gz']
checksums = ['5bc2e164e7cce3bfb93d592df5e3059157c8d510b136535bdb6d09c3ef060f64']
```